CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * [#187](https://github.com/nf-core/ampliseq/issues/187) - Sample ids are in incorrect order in feature-table from PacBio data
 * [#190](https://github.com/nf-core/ampliseq/pull/190) - Template update for nf-core/tools version 1.12
 * [#147](https://github.com/nf-core/ampliseq/issues/147) - Split `make_classifier` in two different processes that can be allocated different resources
+* [#197](https://github.com/nf-core/ampliseq/issues/197) - "--double_primer" does not run cutadapt twice
```

Comment:
Usually all changes should be written in the CHANGELOG, however, only changes relative to the latest release. And because `--double_primer` was just introduced (see above) I think that line 20 should not be added. If you'd like to have  issue 197 mentioned, I'd recommend to add it to line 11.

Reply:
Ok, I understand, I removed it by reverting the commit.